tests/test_debian_9.py

```diff
@@ -409,66 +445,165 @@ def test_compose_1so_trial(self):
         self.compose_file_name = "1so_trial.yaml"
         self.project_name = generate_random_string()
         container_count, rc = self.compose_up()
+        log_json = self.extract_json("so1")
+        output = self.get_container_logs("so1")
         assert rc == 0
         # Wait for containers to be healthy
         assert self.wait_for_containers(container_count)
         # Check Splunkd on all the containers
         assert self.check_splunkd("admin", self.password)
+        # Check ansible version & configs
+        self.check_ansible(output)
+        # Check values in log output
+        self.check_common_keys(log_json, "so")
     
     def test_compose_1so_custombuild(self):
```

Comment:
Can we can parametrize some of these tests to group them together? Some of these tests seem very similar and it seems we could reduce duplicate code.

Reply:
I see. Though these are testing different scenarios on compose so we don't want to merge the test cases.